### DISCUSSION TITLE
dorm 2.5 New UI

### DISCUSSION THREAD

**Discussion by samuel-ibits** [samuel-ibits/dorm#27] [Announcements]
OK Ghys let's check out the new UI for dorm 2.5 

Let's bring in all our critics and corrections we have from now till Monday 25th to do that so we can move to the next stage
here is the test link
https://samuel-ibits.github.io/dorm/index.html

**Comment by samuel-ibits** [comment]
wait i cant see market
@Joe-EtubiGreatattai

**Comment by Tfix1** [comment]
I can't get pass the study tools o 
The navigation bar is not showing

**Comment by Joe-EtubiGreatattai** [comment]
I noticed the navigation menu is not in the screen in mobile view, I'll fix that today,but to inspect other pages, view in desktop mode to use the menu to navigate, also so noticed the full view style sheet of Market is not working, Samuel please check it, to view market grid view first go to market which will first take you to full view then still click market icon to open grid view.

**Reply by samuel-ibits** [reply]
i dont understand

**Comment by samuel-ibits** [comment]
@Joe-EtubiGreatattai to fix the error in market that stops images from loading you have to remove the'/' before that URL
EG
your code: "/images/house%202.jpg"

change to: "images/house%202.jpg"

take note from henceforth best practise
it also applies to styles and js

**Reply by Joe-EtubiGreatattai** [reply]
Yes boss

**Reply by samuel-ibits** [reply]
it also applies to styles and js check everywhere thts why its not loading see market pic are showing now to prove that wat am saying is true

**Comment by samuel-ibits** [comment]
@Joe-EtubiGreatattai  then check the former ui and study what shelman did to make the menu static on mobile

**Comment by samuel-ibits** [comment]
@Joe-EtubiGreatattai  then check the former ui and study what @Tfix1  did to make the menu static on mobile

**Comment by samuel-ibits** [comment]
@Joe-EtubiGreatattai  then check the former UI and study what @Tfix1  did to make the menu static on mobile
